src/main/java/core/basesyntax/controller/ConsoleHandler.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax.controller;
+
+import core.basesyntax.dao.BetDao;
+import core.basesyntax.dao.BetDaoImpl;
+import core.basesyntax.model.Bet;
+import java.util.Scanner;
+
+public class ConsoleHandler {
+    BetDao betDao = new BetDaoImpl();
+
+    public void handle() {
+        Scanner scanner = new Scanner(System.in);
+        while (true) {
+            String command = scanner.nextLine();
+            if (command.equalsIgnoreCase("quit")) {
+                return;
+            }
+            Bet bet = null;
+            try {
+                String[] betData = command.split(", ");
+                int value = Integer.parseInt(betData[0]);
+                double risk = Double.parseDouble(betData[1]);
+                bet = new Bet(value, risk);
```

Comment:
What if there is only one `element` In the `betData`?  It would throw an `ArrayOutOfBoundsException`.

Reply:
handled `ArrayIndexOutOfBoundsException` with system message